README.md

```diff
@@ -94,12 +94,14 @@ const newman = require('newman'); // require newman in your project
 newman.run({
     collection: require('./sample-collection.json'),
     reporters: 'cli'
-}, function (err) {
+}, function (err, data) {
 	if (err) { throw err; }
-    console.log('collection run complete!');
+    console.log('collection run complete! Summary Run Data', data);
```

Comment:
```suggestion
    console.log('collection run complete!');
```

I suggest not including this in the getting started snippet since the `data` object includes all the execution details and in-general very long JSON text. Which I feel will be confusing for the new users.